internal/gcs/guestconnection.go

```diff
@@ -113,6 +113,27 @@ func (gc *GuestConnection) connect(ctx context.Context) (err error) {
 	if gc.os == "" {
 		gc.os = "windows"
 	}
+	if resp.Capabilities.SendHostCreateMessage {
```

Comment:
Good catch. I forgot we opt out of these for LCOW

Reply:
The annoying thing is that we have to send a v1 document to the guest for it to work. Thus the uvmConfig type.